src/test/java/org/reaktivity/nukleus/http2/internal/bench/Http2ServerBM.java

```diff
@@ -94,7 +95,8 @@
             LangUtil.rethrowUnchecked(ex);
         }
 
-        reaktor = Reaktor.launch(configuration, n -> "http".equals(n), Http2Controller.class::isAssignableFrom);
+        //reaktor = Reaktor.launch(configuration, n -> "http".equals(n), Http2Controller.class::isAssignableFrom);
+        reaktor = null;
```

Comment:
It was just copied from Http nukleus, so it is not functional yet. It will be addressed in the future PRs